.github/actions/baas-test-server/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Start BaaS test server
+description: Uses
+inputs:
+  branch:
+    description: BaaS branch to use when starting the server
+    default: master
+  githash:
+    description: Specific githash to use when starting the server (this is used instead of branch if provided)
+outputs:
+  container-id:
+    description: The id of the container created
+  baas-url:
+    description: The url of the BaaS server created
+  mongo-url:
+    description: The url of the MongoDB instance backing the server created
+
+runs:
+  using: node20
```

Comment:
I would actually have liked to just use a composite action here, but it doesn't support specifying a "post" step.